src/Core/Services/Implementations/HandlebarsMailService.cs

```diff
@@ -416,9 +416,20 @@ public async Task SendEnqueuedMailMessageAsync(IMailQueueMessage queueMessage)
         public async Task SendAdminResetPasswordEmailAsync(string email, string userName, string orgName)
         {
             var message = CreateDefaultMessage("Master Password Has Been Changed", email);
+
+            string name;
+            if (string.IsNullOrEmpty(userName))
```

Comment:
I think this logic can be extracted to a private method since it is used more than once. What do you think?

Reply:
Great call, should be updated now!